src/main.ts

```diff
@@ -123,6 +124,42 @@ export default function main() {
     return res;
   }
 
+  type FeedbackOptions = {
+    featureId: string;
+    sentiment: "like" | "dislike";
+    userId?: string;
+    companyId?: string;
+    comment?: string;
+  };
+  async function feedback({
+    featureId,
+    sentiment,
+    userId,
+    companyId,
+    comment,
+  }: FeedbackOptions) {
+    checkKey();
+    if (!featureId) err("No featureId provided");
+
+    const payloadUserId = userId || getSessionUser();
+    if (!payloadUserId) {
+      err("No userId provided and persistUser is disabled");
+      return;
+    }
+
```

Comment:
Given that `sentiment` is a required parameter, we should probably add runtime check for it too, like
```
if (!sentiment) err("No sentiment provided");
```
Should sentiment be required tho? Isn't it possible to add a written feedback (note) without proving a like or dislike?

Reply:
no way to send feedback without a sentiment in this version. We can always make it more lax in future iterations.